python/paddle/fluid/framework.py

```diff
@@ -1763,7 +1769,10 @@ def numpy(self):
         return self.real.numpy() + 1j * self.imag.numpy()
 
     def __str__(self):
-        return "REAL: " + self.real.__str__() + "IMAG: " + self.imag.__str__()
+        return "ComplexTensor[real]: %s\n%s" % (
+            self.real.name, str(self.real.value().get_tensor())
+        ) + "\nComplexTensor[imag]: %s\n%s" % (
+            self.imag.name, str(self.imag.value().get_tensor()))
```

Comment:
Better formatting single str instead of using '+' for several strings.

Reply:
Done